exercises/practice/acronym/acronym.wat

```diff
@@ -1,6 +1,20 @@
+;;
+;; Note to CLI Users: Read 'exercises/shared/.docs/debug.md' for instructions
```

Comment:
Students won't have access to this file without checking out the wasm repo, so I think we should try a different approach.
What I think we should do is to duplicate the contents from `exercises/shared/.docs/debug.md` into `exercises/shared/.docs/help.md`, which is a CLI-only file that ends up getting downloaded to a `HELP.md` document in the downloaded exercise's directory. See https://exercism.org/docs/building/tracks/shared-files#h-file-help-md The duplication isn't great, but if that bothers you too much, we could also create a new document in the `docs` folder and link to that document. But having this inlined is probably better.

Reply:
Sounds like a good approach to me! I removed the prepended block comments in source and appended the contents of `debug.md` to `help.md`